lib/validate.js

```diff
@@ -94,8 +94,8 @@ exports.usingChecksum = (deviceFileDescriptor, options = {}) => {
     imageSize: options.imageSize,
     progress: options.progress,
     chunkSize: options.chunkSize
-  }).then((deviceChecksum) => {
-    if (options.imageChecksum !== deviceChecksum) {
+  }).then((results) => {
+    if (options.imageChecksum !== results.crc32) {
```

Comment:
> Could we ask the user to pass the checksum type as well?

Yeah, that's seems like a good way to do it, but doesn't consider the case of calculating multiple checksums.
Maybe we could make `.imageChecksum` and object as well, like
```js
{
  imageChecksum: {
    crc32: '...',
    md5: '...',
    sha256: '...'
  }
}
```
and then check against all of them
```js
for( var k in imageChecksum ) {
  if( results[k] !== imageChecksum[k] )
    throw errors.ValidationError
}
```
through which we could actually also default the `.algorithms` option to `Object.keys(imageChecksum)` if not explicitly set.
What do you think?